src/main/scala/uk/gov/hmrc/hmrcfrontend/views/helpers/MessageHelpers.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.hmrcfrontend.views.helpers
+
+import play.api.i18n.{Lang, Messages, MessagesApi, MessagesImpl}
+
+object MessageHelpers {
+  def messagesForLanguage(
+    code: Option[String] = None
+  )(implicit messages: Messages, messagesApi: MessagesApi): Messages =
+    // TODO: PR question: should I make this a class and @inject messagesApi and pass helper via DI?
+    // TODO: PR question: should I require lang supplied be supported / print a warning / solely rely on messagesApi fallback behaviour?
```

Comment:
I probably favour raising an exception if the lang isn't in `play.api.Langs.availables` but didn't implement because with current implementation it would need another implicit, but if I refactor and to @inject instead then it's easy to add